shell/pages/c/_cluster/apps/charts/index.vue

```diff
@@ -262,6 +270,18 @@ export default {
 
     clusterId() {
       return this.$store.getters['clusterId'];
+    },
+
+    noFiltersApplied() {
+      return Object.values(this.internalFilters).every((arr) => arr.length === 0) && !this.searchQuery;
+    },
+
+    totalMessage() {
```

Comment:
this will update before the results of the filter are shown

- no filter - `253 charts in total`
- click on a filter - `253 charts matching your criteria.`
- list updated - `21 charts matching your criteria.`

the number should be aligned with what we're showing. i'm guessing this is rendering lag